frontend/src/dataUtilities.ts

```diff
@@ -464,6 +491,60 @@ export const incomePaystubsCategories: Array<Categories> = [
   },
 ];
 
+
+export const checkReportGetCategories: Array<Categories> = [
```

Comment:
"Check Report" (aka "Consumer Report") is the name for this category of products.
"Base Report" is the name of the product itself, so variable names should correspond to Base Report.

With that in mind, can you update this to `checkReportBaseReportGet...` to avoid confusion?